cap/modules/repoimporter/utils.py

```diff
@@ -45,9 +45,11 @@
                      r'[:|\/](?P<owner>[a-zA-Z][\w.-]+)\/(?P<repo>[\w.-]+)' \
                      r'/tree/(?P<branch>.+)'
 
-GIT_REG_FOR_FILE = r'(?P<host>https://[github\.com|gitlab\.cern\.ch]+)[:|\/]' \
-                     r'(?P<owner>[a-zA-Z][\w.-]+)\/(?P<repo>[\w.-]+)' \
-                     r'/blob/(?P<branch>[-a-zA-Z]+)/(?P<filepath>.+)'
+GIT_REG_FOR_FILE = r'(?P<host>https://[github\.com|gitlab\.cern\.ch' \
+                   r'|gitlab\-test\.cern\.ch]+)[:|\/]' \
+                   r'(?P<owner>[a-zA-Z][\w.-]+)\/' \
+                   r'(?P<repo>[\w.-]+)/blob/' \
+                   r'(?P<branch>[-a-zA-Z]+)/(?P<filepath>.+)'
 
 
 def parse_url(url):
```

Comment:
as we discussed, this will be incorrect in some cases 
need some small fixes in logic ;)